span-processing-config-service-impl/src/main/java/org/hypertrace/span/processing/config/service/SpanProcessingConfigServiceImpl.java

```diff
@@ -288,6 +295,45 @@ public void updateApiNamingRule(
     }
   }
 
+  @Override
+  public void updateApiNamingRules(
+      UpdateApiNamingRulesRequest request,
+      StreamObserver<UpdateApiNamingRulesResponse> responseObserver) {
+    try {
+      RequestContext requestContext = RequestContext.CURRENT.get();
+      this.validator.validateOrThrow(requestContext, request);
+
+      List<UpdateApiNamingRule> updateApiNamingRules = request.getRulesList();
+      List<ApiNamingRule> existingRules =
```

Comment:
Do we not have bulk get in config store?

Reply:
No Shubham.🥲